deb/common/control

```diff
@@ -61,6 +60,7 @@ Description: Docker: the open-source application container engine
 Package: docker-ce-cli
 Architecture: linux-any
 Depends: ${shlibs:Depends}
+Recommends: git, procps
```

Comment:
Shouldn't `procps` be in the `docker-ce` package above (not in the cli)?

Reply:
Oh right -- I got confused because `docker top` is a CLI command, but it really runs `ps` from the daemon context; will update!